app/helpers/abr_pdf_fields.rb

```diff
@@ -123,6 +123,53 @@ def delivery_city_state_zip
     "#{delivery_city}, #{delivery_state} #{delivery_zip}"
   end
   
+  def pdf_cover_fields
+    [
+      "SUBMIT YOUR COMPLETED SIGNED FORM TO",
+      "Request Deadline",
+      "Reminders for your state",
+    ]
+  end
+
+  def submit_your_completed_signed_form_to
+    "\n#{state_registrar_address_newlines}"
+  end
+ 
+  def request_deadline 
+    "\n#{self.cover_state_deadline}"
+  end
+
+  def reminders_for_your_state 
+    "#{self.cover_state_reminders}"
+  end
+
+
+  def state_registrar_address_newlines
```

Comment:
You should be able to use the above `delivery_full_address` instead of having to re-convert from the html.

Reply:
I'm not getting that reliably.

![image](https://user-images.githubusercontent.com/5473291/96934037-8b9af800-148f-11eb-8422-062a5efd4b33.png)

Reply:
The delivery_full_address code looks like it is trying to get it as an attribute of the registrar address -- which is a string, so that wouldn't work.

![image](https://user-images.githubusercontent.com/5473291/96934221-db79bf00-148f-11eb-8bf4-777bd1ef4c5e.png)

Reply:
I noted this in slack, but `delivery_full_address`  going through `state_registrar_office`  and then `abr_office`  only has access to the data from calls through `ZipCodeCountyAddress.where(:zip=>zip_code).first` 

Conversely  `state_registrar_address`, going through `abr_address` ends up falling back to attributes attached directly to the state object, i.e.,  `registrar_address` and `registrar_abr_address`  and those are actually stored natively as HTML. 

So, I think for these purposes, I'll keep the current code, but the above should be good to know re: delivery_full_address not being equivalent.